### ISSUE TITLE
terragrunt asks to overwrite remote state config every run

### ISSUE THREAD

**Issue by bl1nk** [gruntwork-io/terragrunt#188]
Hey there,
I just migrated to terragrunt 0.12.x and terraform 0.9 with s3 remote states and locking in dynamodb.

Every time I run terragrunt it asks me to overwrite the remote state configuration.
```
$ tg refresh
[terragrunt] [/Users/x/x/terraform/terraform-x] 2017/04/25 13:11:21 Running command: terraform --version
[terragrunt] 2017/04/25 13:11:21 Reading Terragrunt config file at /Users/x/x/terraform/terraform-x/terraform.tfvars
[terragrunt]  WARNING: Terraform remote state is already configured for backend s3 with config map[region:eu-west-1 bucket:x-terraform encrypt:true key:terraform-x/terraform.tfstate lock_table:terraform-lock], but your Terragrunt configuration specifies config map[encrypt:true key:terraform-x/terraform.tfstate region:eu-west-1 bucket:x-terraform lock_table:terraform-lock]. Overwrite? (y/n) y
```
It seems the map is always in a different order. Does not matter what command I run.

`terraform.tfvars` in my project
```
terragrunt = {
    include {
        path = "${find_in_parent_folders()}"
    }
}
```
Backend is defined empty in the `main.tf`
```
terraform {
    backend "s3" {}
}
```

And finally my `terraform.tfvars` in the root terraform directory
```
terragrunt = {
    remote_state {
        backend = "s3"
        config {
            encrypt = true
            key = "${path_relative_to_include()}/terraform.tfstate"
            region = "eu-west-1"
            bucket = "x-terraform"
            lock_table = "terraform-lock"
        }
    }
}
```

**Comment by brikis98** [comment]
Definitely a bug. Looks like a side effect from https://github.com/gruntwork-io/terragrunt/pull/181.

**Comment by brikis98** [comment]
This should be fixed by #189. Check out https://github.com/gruntwork-io/terragrunt/releases/tag/v0.12.9.
